<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(1, 10, 19);
background-color: rgb(1, 10, 19);
border-color: rgb(120, 90, 40);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="SUMMONERNAME">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
border-color: rgb(120, 90, 40);
background-color: rgb(0, 3, 6);
border: 1px solid rgb(120, 90, 40);
color: rgb(120, 90, 40);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="autoFormatting">
     <set>QTextEdit::AutoNone</set>
    </property>
    <property name="placeholderText">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
color: rgb(120, 90, 40);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Summoner Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
color: rgb(120, 90, 40);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Season:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SEASONS">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>125</red>
          <green>124</green>
          <blue>114</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>16</red>
          <green>23</green>
          <blue>31</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(16, 23, 31);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);
</string>
    </property>
    <item>
     <property name="text">
      <string>SEASON3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEASON2014</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEASON2015</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEASON2016</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SEASON2017</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(16, 23, 31);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);

</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dataBackground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1121</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(16, 23, 31);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Parse</string>
    </property>
   </widget>
   <widget class="QListWidget" name="champList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(1, 10, 19);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);
</string>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="champsLabelTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Narkisim&quot;;
color: rgb(120, 90, 40);
border-color: rgb(120, 90, 40);
border: 1px solid rgb(120, 90, 40);
background-color: rgba(1, 10, 19, 100);</string>
    </property>
    <property name="text">
     <string>Champions</string>
    </property>
   </widget>
   <widget class="QFrame" name="overlayBackground">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1121</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(16, 23, 31);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="champNAME">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;Narkisim&quot;;
background-color: rgba(1, 10, 19, 100);
color: rgb(120, 90, 40);
border-color: rgb(120, 90, 40);
border: 1px solid rgb(120, 90, 40);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dataDisplay">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>831</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
background-color: rgb(1, 10, 19);
border-color: rgb(120, 90, 40);
color: rgb(125, 124, 114);
border: 1px solid rgb(120, 90, 40);
</string>
    </property>
   </widget>
   <widget class="QFrame" name="champImgIcon">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>160</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(1, 10, 19, 100);
border-color: rgb(120, 90, 40);
border: 1px solid rgb(120, 90, 40);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="champSearch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Narkisim&quot;;
border-color: rgb(120, 90, 40);
background-color: rgb(0, 3, 6);
border: 1px solid rgb(120, 90, 40);
color: rgb(120, 90, 40);
</string>
    </property>
    <property name="placeholderText">
     <string>Search Champions</string>
    </property>
   </widget>
   <zorder>pushButton_2</zorder>
   <zorder>dataBackground</zorder>
   <zorder>overlayBackground</zorder>
   <zorder>SUMMONERNAME</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>SEASONS</zorder>
   <zorder>pushButton</zorder>
   <zorder>champList</zorder>
   <zorder>champsLabelTitle</zorder>
   <zorder>champNAME</zorder>
   <zorder>dataDisplay</zorder>
   <zorder>champImgIcon</zorder>
   <zorder>champSearch</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
